src/Projektanker.Icons.Avalonia/Icon.axaml.cs

```diff
@@ -35,6 +36,23 @@ public DrawingImage DrawingImage
             private set => SetAndRaise(DrawingImageProperty, ref _drawingImage, value);
         }
 
+        public IBitmap BitmapImage
+        {
+            get
+            {
+                if (_drawingImage is null) return null;
+                var renderBitmap = new RenderTargetBitmap(new PixelSize((int) _drawingImage.Size.Width, (int) _drawingImage.Size.Height));
+                
+                using (var ctx = renderBitmap.CreateDrawingContext(null))
+                using (var dtx = new DrawingContext(ctx))
+                {
+                    _drawingImage.Drawing.Draw(dtx);
+                }
+
+                return renderBitmap;
+            }
+        }
+
```

Comment:
Should be removed as well.

Reply:
That's just an utility method with no penalty, and was the best way i find to convert the drawing to bitmap.
But it's your call :)